static/src/javascripts/projects/common/modules/ui/first-pv-consent-banner.spec.js

```diff
@@ -111,6 +119,16 @@ describe('First PV consents banner', () => {
             });
             return expect(await banner.canShow()).toBe(false);
         });
+        it('should render outside the EU if in test', async () => {
```

Comment:
Minor: it would be more specific to mention the test name and/or variant

```js
it('should render outside the EU when commercial consent test participation is "variant"', () => {
```